symplyphysics/core/symbols/wrappers.py

```diff
@@ -0,0 +1,55 @@
+from typing import Self, Any
+from sympy import Symbol as SymSymbol
+from .symbols import HasDimension, next_name
+
+
+class Wrapper(HasDimension, SymSymbol):
```

Comment:
Too generic name. I'm thinking about `Operation` but how is it different from `Function`? Maybe try to use existing `Function` implementation to cover `Average` functionality?

Reply:
I think there are some common points between this and `Function` (e.g. they accept an argument, although functions can accept more than one unlike these `Wrappers`), but they're different in that a `Function` always wraps its argument(s) in parentheses in both plain code and LaTeX, and in several frequent cases we need something with a custom "wrapping" functionality. Within plain code though they do look like functions (`avg(K)`, `Delta(T)`, `d(S)`, etc), but in LaTeX that's not always the case ($`\langle K \rangle`$ vs $`\Delta T`$ and $`dS`$).

I agree this functionality can be put in the `operations` folder.

Reply:
I think we can subclass from `Function` but override printing methods to allow custom display.

Reply:
Okay, I'll give it a try.